### ISSUE TITLE
Missing i386 libraries

### ISSUE THREAD

**Issue by topas-rec** [aljordan/DRCDesignerLinux#1]
Hello @aljordan ,

I use this since a couple of years with some libraries missing on a Debian system. Util now I was able to get it working with some investigation. Iinstalling the libraries in the i386 architecture and use it on a amd64 architecture.
I think you mentioned that DRCDesigner only runs on x86 systems, but I was able to get good results on x64 OS, too.

Currently on Debian 10.10 I run into the following missing dependency:
```
ALSA lib conf.c:3558:(snd_config_hooks_call) Cannot open shared library libasound_module_conf_pulse.so (/usr/lib/i386-linux-gnu/alsa-lib/libasound_module_conf_pulse.so: libasound_module_conf_pulse.so: cannot open shared object file: No such file or directory)
```

Perhaps I get this working soon, but I want to ask if there is a way to improve this and make the program work on x64 OS?
Can you point out a way on how to do this?

Regards Tobias

**Comment by topas-rec** [comment]
I found https://github.com/aljordan/DRCDesignerLinux/blob/108c1ff82ab48f71e8e824c1c0145794ebdfc1ad/DRCDesignerLinux/rec_imp/ImportantNoteRegarding64BitLinuxSystems.txt
and it shows that rec_imp is the cause for all the trouble:
```
Because rec_imp was compiled for 32 bit systems, it will fail on modern 64 bit Linux systems.
In order to get it to run on 64 bit linux systems, extra packages will have to be installed.

Below are the packages needed to run rec_imp on 64 bit Ubuntu 14.04 variations:
libasound2:i386, libjack0:i386, lib32stdc++6 lib32z1, lib32z1-dev, libasound2-plugins:i386

Install them with the following commands:
sudo apt-get install libasound2:i386
sudo apt-get install libjack0:i386
sudo apt-get install lib32stdc++6 lib32z1 lib32z1-dev
sudo apt-get install libasound2-plugins:i386
```

I currently cannot install `libasound2:i386` because:
```
$ sudo apt-get install libasound2-plugins:i386
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libasound2-plugins:i386 : Depends: libpulse0:i386 (>= 0.99.1) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```

I'll try to solve this.
A real solution would be to compile rec_imp for 64 bit systems. Right?

**Comment by topas-rec** [comment]
I was able to compile the executable `rec_imp` on Debian 10.10 and replacing the executable in the DRCDesigner folder resolves the dependency problems and makes DRCDesigner list all audio interfaces of my pc again. I don't know if it works as it should and we should carefully look at it cause things like optimizations have been disabled to make floating point operations calculate things exactly.

I added an include in a file and ignored some warnings regarding deprecated `jack` api functions.

I plan to create a pull request.
